app/_includes/footer.html

```diff
@@ -1,57 +1,121 @@
+        <div class="social">
+
+          <div class="social-link">
+            <a href="https://www.facebook.com/konghq/" title="Facebook"
+              ><i
+                aria-label="Facebook"
+                class="fa fa-facebook-official"
+                aria-hidden="true"
+              ></i
+            ></a>
+          </div>
+          <div class="social-link">
+            <a href="https://twitter.com/thekonginc" title="Twitter"
+              ><i
+                aria-label="Twitter"
+                class="fa fa-twitter"
+                aria-hidden="true"
+              ></i
+            ></a>
+          </div>
+          <div class="social-link">
+            <a href="https://www.meetup.com/topics/kong/all/" title="Meetup"
+              ><i
+              aria-label="Meetup"
+              class="fa fa-meetup"
+              aria-hidden="true"
+            ></i
+          ></a>
+          </div>
+          <div class="social-link">
+            <a href="https://linkedin.com/company/278819" title="LinkedIn"
+              ><i aria-label="GitHub" class="fa fa-linkedin" aria-hidden="true"></i
+            ></a>
+          </div>
+          <div class="social-link">
+            <a href="https://github.com/kong/kong" target="_blank" class="btn-gh">
+              <i class="fa fa-github" aria-hidden="true" aria-label="GitHub"></i> Star
+            </a>
+          </div>
         </div>
       </li>
       <li>
         <nav>
-          <h4>Company</h4>
+          <h5>Products</h5>
           <ul>
-            <li><a href="https://konghq.com/about-kong-inc/">About</a></li>
-            <li><a href="https://konghq.com/investors/">Investors</a></li>
-            <li><a href="http://konghq.com/blog/">Blog</a></li>
-            <li><a href="https://konghq.com/jobs/">Jobs</a></li>
-            <li><a href="https://konghq.com/contact/">Contact</a></li>
-            <li><a href="https://support.konghq.com">Support</a></li>
+            <li><a href="http://konghq.com/kong/">Kong</a></li>
+            <li><a href="http://konghq.com/kong-enterprise/">Kong Enterprise</a></li>
+            <li><a href="http://konghq.com/cloud/">Kong Cloud</a></li>
+            <li><a href="http://konghq.com/subscriptions/">Subscriptions</a></li>
           </ul>
         </nav>
       </li>
       <li>
         <nav>
-          <h4>Products</h4>
+          <h5>Resources</h5>
           <ul>
-            <li><a href="https://docs.konghq.com/">Kong</a></li>
-            <li><a href="https://konghq.com/api-dev-portal/">Dev Portal</a></li>
-            <li><a href="https://konghq.com/api-analytics/">API Analytics</a></li>
-            <li><a href="https://konghq.com/kong-enterprise-edition/">Enterprise</a></li>
+            <li>
+              <a href="https://support.konghq.com" target="_blank"
+                >Enterprise Support</a
+              >
+            </li>
+            <li>
+              <a href="https://docs.konghq.com" target="_blank"
```

Comment:
Is there a reason these two open new tabs, but the old ones don't?

Reply:
yeah the reason is it's a clone of konghq.com on which this is the intended action- good catch